### ISSUE TITLE
[Help] 使用广告数据集，CPU部署训练时train.sh被killed掉

### ISSUE THREAD

**Issue by jinchao-ai** [THUDM/ChatGLM-6B#589]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

[INFO|modeling_utils.py:2693] 2023-04-13 17:04:58,755 >> Generation config file not found, using a generation config created from the model config.
input_ids [5, 65421, 61, 67329, 32, 98339, 61, 72043, 32, 65347, 61, 70872, 32, 69768, 61, 68944, 32, 67329, 64103, 61, 96914, 130001, 130004, 5, 87052, 96914, 81471, 64562, 65759, 64493, 64988, 6, 65840, 65388, 74531, 63825, 75786, 64009, 63823, 65626, 63882, 64619, 65388, 6, 64480, 65604, 85646, 110945, 10, 64089, 65966, 87052, 67329, 65544, 6, 71964, 70533, 64417, 63862, 89978, 63991, 63823, 77284, 88473, 64219, 63848, 112012, 6, 71231, 65099, 71252, 66800, 85768, 64566, 64338, 100323, 75469, 63823, 117317, 64218, 64257, 64051, 74197, 6, 63893, 130005, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3]
inputs 类型#裤*版型#宽松*风格#性感*图案#线条*裤型#阔腿裤 宽松的阔腿裤这两年真的吸粉不少,明星时尚达人的心头爱。毕竟好穿时尚,谁都能穿出腿长2米的效果宽松的裤腿,当然是遮肉小能手啊。上身随性自然不拘束,面料亲肤舒适贴身体验感棒棒哒。系带部分增加设计看点,还
label_ids [-100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, 130004, 5, 87052, 96914, 81471, 64562, 65759, 64493, 64988, 6, 65840, 65388, 74531, 63825, 75786, 64009, 63823, 65626, 63882, 64619, 65388, 6, 64480, 65604, 85646, 110945, 10, 64089, 65966, 87052, 67329, 65544, 6, 71964, 70533, 64417, 63862, 89978, 63991, 63823, 77284, 88473, 64219, 63848, 112012, 6, 71231, 65099, 71252, 66800, 85768, 64566, 64338, 100323, 75469, 63823, 117317, 64218, 64257, 64051, 74197, 6, 63893, 130005, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100, -100]
labels 宽松的阔腿裤这两年真的吸粉不少,明星时尚达人的心头爱。毕竟好穿时尚,谁都能穿出腿长2米的效果宽松的裤腿,当然是遮肉小能手啊。上身随性自然不拘束,面料亲肤舒适贴身体验感棒棒哒。系带部分增加设计看点,还
/usr/local/python3/lib/python3.7/site-packages/transformers/optimization.py:395: FutureWarning: This implementation of AdamW is deprecated and will be removed in a future version. Use the PyTorch implementation torch.optim.AdamW instead, or set `no_deprecation_warning=True` to disable this warning
  FutureWarning,

  **### **0%|                                                                                     | 0/2000 [00:00<?, ?it/s]****

04/13/2023 17:06:58 - WARNING - transformers_modules.chatglm-6b.modeling_chatglm - `use_cache=True` is incompatible with gradient checkpointing. Setting `use_cache=False`...
./train.sh: line 24:  4809 Killed                  CUDA_VISIBLE_DEVICES=0 python3 main.py --do_train --train_file AdvertiseGen/train.json --validation_file AdvertiseGen/dev.json --prompt_column content --response_column summary --overwrite_cache --model_name_or_path /run/xunyin/chatglm-6b --output_dir output/adgen-chatglm-6b-pt-$PRE_SEQ_LEN-$LR --overwrite_output_dir --max_source_length 64 --max_target_length 64 --per_device_train_batch_size 1 --per_device_eval_batch_size 1 --gradient_accumulation_steps 16 --predict_with_generate --max_steps 2000 --logging_steps 10 --save_steps 500 --learning_rate $LR

### Expected Behavior

参考文档：https://github.com/THUDM/ChatGLM-6B/blob/main/ptuning/README.md

下载的AdvertiseGen数据集已放在ptuning目录下。
部署方式：CPU部署，没有GPU是不是不能train？
train.sh去掉了最后两行
#    --pre_seq_len $PRE_SEQ_LEN \
#    --quantization_bit 4

脚本执行到黑色加粗字体时，卡住，一段时间后，sh脚本被killed，也没有报错信息

正常train成功，应该在output目录下面有train好的模型吧

### Steps To Reproduce

1. git clone https://github.com/THUDM/ChatGLM-6B
2. 删除 train.sh最后两行配置，指定model_name_or_path路径为绝对路径（git clone https://huggingface.co/THUDM/chatglm-6b，所在位置）

3. bash train.sh

### Environment

```markdown
- OS:CentOS 7.4
- Python:3.7.5
- Transformers:4.27.1
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by twosnowman** [comment]
same problem(I'm using Nvidia 3060 12G), did you solve it now?
